src/gennylib/include/gennylib/context.hpp

```diff
@@ -610,24 +612,24 @@ class PhaseContext final : public v1::HasNode {
         if (auto metricsName = this->_node["MetricsName"].maybe<std::string>()) {
             stm << *metricsName;
         } else {
-            stm << defaultMetricsName << "." << _phaseNumber;
+            stm << defaultMetricsName << "." << this->_phaseNumber;
         }
 
         return this->workload()._registry.operation(
-            this->_actor->operator[]("Name").to<std::string>(),
+            this->_actorContext->operator[]("Name").to<std::string>(),
             stm.str(),
             id,
-            _phaseNumber,
+            this->_phaseNumber,
             internal);
     }
 
     /**
      * @brief [Experimental] Create a metrics::Operation with the specified name
      *
-     * The current operation() function will always return the same operation if MetricsName is specified.
-     * According to the HelloWorld actor, it seems not the expected behavior
-     * But it is too risky to change the behavior without thoroughly exam the impact
-     * Created TIG-4328 to track this work
+     * The current operation() function will always return the same operation if MetricsName is
+     * specified. According to the HelloWorld actor, it seems not the expected behavior But it is
```

Comment:
This comment has not been improved by auto-formatting

Reply:
I agree. I've fixed this comment.